sources/packages/backend/libs/integrations/src/esdc-integration/student-loan-balances/student-loan-balances-files/student-loan-balances-file-response.ts

```diff
@@ -38,9 +38,10 @@ export class StudentLoanBalancesFileResponse {
     return getDateOnlyFromFormat(this.line.substring(91, 99), DATE_FORMAT);
   }
   /**
-   * Date of birth of the student to match and update the student loan balance.
+   * CSL balance. Expected 7 characters where the last 2 positions should be considered decimals.
    */
   get cslBalance(): number {
-    return +this.line.substring(13, 20);
+    // Divide by 100 to convert to 2 decimal places.
+    return +this.line.substring(13, 20) / 100;
```

Comment:
Can we please add a TODO here to centralize this parse decimal logic ?. (It is already in sfas utils. we could move it to common place when refactoring).

Reply:
I saw it on SFAS but did not felt worth enough to have a util method.

Reply:
Added TODO to move on.